src/common/mfu_flist_copy.c

```diff
@@ -330,18 +332,44 @@ static int mfu_copy_xattrs(
     /* iterate over list and copy values to new object lgetxattr/lsetxattr */
     if(got_list) {
         char* name = list;
+        struct error_context ctx;
 
         while(name < list + list_size) {
             /* start with a reasonable buffer,
              * allocate something bigger as needed */
             size_t val_bufsize = 1024;
             void* val = (void*) MFU_MALLOC(val_bufsize);
+            int copy_xattr = 1;
 
             /* lookup value for name */
             ssize_t val_size;
             int got_val = 0;
 
-            while(! got_val) {
+            if (copy_opts->copy_xattrs == XATTR_COPY_ALL) {
+                copy_xattr = 1;
+            } else if (copy_opts->copy_xattrs == XATTR_COPY_NONE) {
+                copy_xattr = 0;
+            } else if (copy_opts->copy_xattrs == XATTR_USE_LIBATTR) {
+                if (attr_copy_action(name, &ctx) == ATTR_ACTION_SKIP)
+                    copy_xattr = 0;
```

Comment:
Coding style: we try to always use braces, even for one-liners.
```suggestion
                if (attr_copy_action(name, &ctx) == ATTR_ACTION_SKIP) {
                    copy_xattr = 0;
                }
```

Reply:
Done